### DISCUSSION TITLE
How do I use a break tag? (<br>)

### DISCUSSION THREAD

**Discussion by guifeliper** [amannn/next-intl#427] [Q&A] [answered]
Hello guys, 
I am trying to add a brake tag to my intl 
My current translation is something like the following: 
`"welcome": "Hello {name}. <br/> <strong>How are you?</strong>"`

I would imagine the implementation would be something like 

 ```
    {t.rich("welcome", {
            locale,
            name: person.name,
            strong: (children) => <strong>{children}</strong>,
            br: () => <br />,
          })}
```

Can you help me to implement a brake tag?

**Comment by amannn** [comment, marked as answer]
The ICU parser doesn't support self-closing tags at this point, therefore you have to use `<br></br>`.

[Working example](https://codesandbox.io/s/use-intl-br-example-qn657t?file=/src/Root.tsx:105-172)

I've also added a note about this in [the rich text docs](https://next-intl-docs.vercel.app/docs/usage/messages#rich-text). Thank you for the great question!

**Reply by guifeliper** [reply]
Thank you so much! This works!
